controller/LoginController.java

```diff
@@ -0,0 +1,55 @@
+package controller;
+
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.MenuItem;
+import javafx.scene.control.PasswordField;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+
+public class LoginController {
+
+    @FXML
+    private AnchorPane loginscene;
+
+    @FXML
+    private TextField userinput;
+
+    @FXML
+    private PasswordField passswordinput;
+
+    @FXML
+    private Button loginbutton;
+
+    @FXML
+    private MenuItem gottoHelpCenter1;
```

Comment:
what is this?

Reply:
an ancient relic from the time bevor @TomxHWRxWI20C  toke over
can be deleted